ccm_web/client/src/pages/Home.tsx

```diff
@@ -18,48 +22,101 @@ const useStyles = makeStyles((theme) => ({
   },
   featureCardContainer: {
     padding: 5
+  },
+  courseName: {
+    textAlign: 'left',
+    padding: 5
   }
 }))
 
-function FeatureGroup (featureGroup: FeatureUIGroup): JSX.Element {
-  const classes = useStyles()
-  return (
-    <Grid key={featureGroup.id} container item xs={12} spacing={0}>
-      <Grid item xs={12}><Typography variant='h6' className={classes.title} >{featureGroup.title}</Typography></Grid>
-      <Grid container item xs={12}>
-        {featureGroup.features.sort((a, b) => (a.data.ordinality < b.data.ordinality) ? -1 : 1).map(feature => {
-          return (
-            <Grid key={feature.data.id} item className={classes.featureCardContainer} xs={12} sm={4}>
-              <FeatureCard {...feature} />
-            </Grid>
-          )
-        })}
-      </Grid>
-    </Grid>
-  )
+interface HomeProps extends LtiProps {
+  ltiKey: string | undefined
 }
 
-function Home (): JSX.Element {
+function Home (props: HomeProps): JSX.Element {
+  const [globals, isAuthenticated, loading, error] = useGlobals(props.ltiKey)
   const classes = useStyles()
-  const saveCourseName = async (courseName: string): Promise<void> => await new Promise<void>((resolve, reject) => {
-    console.log('saveCourseName ' + courseName)
-    if (courseName === 'reject') {
-      reject(new Error('Rejected as requested'))
+
+  const renderFeature = (feature: FeatureUIProps): JSX.Element => {
+    return (
+      <Grid key={feature.data.id} item className={classes.featureCardContainer} xs={12} sm={4}>
+        <FeatureCard {...feature} />
+      </Grid>
+    )
+  }
+
+  const renderFeatureGroup = (featureGroup: FeatureUIGroup): JSX.Element => {
+    return (
+      <Grid key={featureGroup.id} container item xs={12} spacing={0}>
+        <Grid item xs={12}><Typography variant='h6' className={classes.title} >{featureGroup.title}</Typography></Grid>
```

Comment:
I think there's a way we can ensure the header is rendered as a `h1` in the HTML but maintains the `h6` style. I'll check the API, might be the `as` prop.

Reply:
It's `component`, see https://material-ui.com/api/typography/#props. I think we want these section headers to be `h2` but have `h6` styling and the course name to be `h1` and have `h5` styling.